atom/browser/common_web_contents_delegate.cc

```diff
@@ -333,13 +335,23 @@ void CommonWebContentsDelegate::DevToolsSaveToFile(
     file_dialog::DialogSettings settings;
     settings.parent_window = owner_window();
     settings.title = url;
-    settings.default_path = base::FilePath::FromUTF8Unsafe(url);
-    if (!file_dialog::ShowSaveDialog(settings, &path)) {
+    PathService::Get(chrome::DIR_DEFAULT_DOWNLOADS, &settings.default_path);
+    settings.default_path =
+      settings.default_path.Append(base::FilePath::FromUTF8Unsafe(url));
+    settings.type = ui::SelectFileDialog::SELECT_SAVEAS_FILE;
+    std::vector<base::FilePath::StringType> extensions;
+    extensions.push_back(
```

Comment:
why json?

Reply:
because the performance profile will be saved as json
<img width="359" alt="screen shot 2017-12-28 at 1 55 22 pm" src="https://user-images.githubusercontent.com/11330831/34424215-d7f9ef7e-ebd6-11e7-98e9-bacb149ff86d.png">

Reply:
I don't think this should be specific to the performance profile

Reply:
I will remove it.